advanced/Scripts/update.sh

```diff
@@ -47,23 +35,29 @@ is_repo() {
 	# Use git to check if directory is currently under VCS
 	echo -n ":::    Checking $1 is a repo..."
 	cd "${1}" &> /dev/null || return 1
-	git status &> /dev/null && echo " OK!"; return 0 || echo " not found!"; return 1
+	if [[ $(git status --short) ]]; then
+    echo " OK!"
+    return 0
+  else
+    echo " not found!"
+    return 1
+  fi
 }
 
 make_repo() {
 	# Remove the non-repod interface and clone the interface
 	echo -n ":::    Cloning $2 into $1..."
 	rm -rf "${1}"
-	git clone -q --depth 1 "${2}" "${1}" > /dev/null & spinner $!
+	git clone -q --depth 1 "${2}" "${1}" > /dev/null || exit 1
```

Comment:
Cool, can you pull this branch and add your additions to it, and push back to this branch? I'd appreciate it.